packages/wonder-stuff-core/src/__tests__/errors-from-error.test.ts

```diff
@@ -31,7 +32,6 @@ describe("#errorsFromError", () => {
             );
 
             // Assert
-            // @ts-expect-error [FEI-5011] - TS2339 - Property 'toBeEmpty' does not exist on type 'JestMatchers<unknown[]>'.
```

Comment:
**praise:** thanks for fixing the underlying issue and removing these suppressions.